hedera-node/src/main/java/com/hedera/services/store/models/TokenRelationship.java

```diff
@@ -208,12 +212,55 @@ private boolean isTokenKycGrantedFor() {
 		return !token.hasKycKey() || kycGranted;
 	}
 
+	public long getKey() {
```

Comment:
Probably preferable to use `EntityNumPair` here, to reduce technical debt if/when we need to change the internal representation of an `(accountId, tokenId)` pair. (E.g. if implementing sharding.)